src/main/java/RecrBad.java

```diff
@@ -0,0 +1,186 @@
+            moreTasks = addDeadlineTask(startIndexOfDescription, line, tasks);
+        } else if (req[0].equalsIgnoreCase("EVENT")) {
+            moreTasks = addEventTask(startIndexOfDescription, line, tasks);
+        } else {
+            // request does not start with todo, deadline, or event
+            throw new InvalidParamsException("No such command." + System.lineSeparator() + PrintHelper.printCommandsList());
+        }
+
+        displayListItem(moreTasks, tasks.length);
+        System.out.println(System.lineSeparator() + "Congrats, now have " + moreTasks.length + " tasks");
+
+        return moreTasks;
+    }
+
+    /**
+     * Prints each Task in Task array
+     *
+     * @param tasks Existing Task array
+     */
+    private static void displayList(Task[] tasks) {
+        int count = 1;
+        for (Task task : tasks) {
+            System.out.println(count + ".[" + task.getType() + "]["
+                    + task.getStatus() + "] " + task.getDescription());
+            count += 1;
+        }
+    }
+
+    /**
+     * Prints a specific task in Tasks array
+     *
+     * @param tasks existing Task array
+     * @param index index of specific task
+     */
+    private static void displayListItem(Task[] tasks, int index) {
+        System.out.println("[" + tasks[index].getType() + "]["
+                + tasks[index].getStatus() + "] "
+                + tasks[index].getDescription());
+    }
+
+    /**
+     * Marks Task as done or not done
+     *
+     * @param tasks  existing Tasks array
+     * @param req    String[] input from user
+     * @param isMark type of operation: mark or unmark
+     */
+    private static void markOperation(Task[] tasks, String[] req, boolean isMark) {
+        if (req.length < 1) {
+            System.out.println("invalid mark/ unmark operation");
+            return;
+        }
+        int taskNum = Integer.parseInt(req[1]);
+        if (tasks.length < taskNum) {
+            System.out.println("No such taskNum");
+            return;
+        }
+        if (isMark) { // MARK #taskNum as done
+            tasks[taskNum - 1].markAsDone();
+            System.out.println("Has marked task" + taskNum + ":");
+        } else {
+            tasks[taskNum - 1].markAsNotDone();
+            System.out.println("Has unmarked task" + taskNum + ":");
+        }
+        displayListItem(tasks, taskNum - 1);
+    }
+
+    /**
+     * Input
+     * [any text]           to addToList,
+     * list                 to displayList,
+     * mark/unmark [index]  to mark item
+     */
+    public static void main(String[] args) { // Ctrl B to see def, shift F10 to run, Ctrl Alt L reformat
+        Scanner in = new Scanner(System.in);
+        PrintHelper.sayHi();
+        Task[] tasks = new Task[]{};
+
+        while (true) {
+            PrintHelper.printLine();
+            String line = in.nextLine(); // reads input
+            String[] req = line.split(" ");
```

Comment:
`req` could be better named, it is not immediately obvious to the reader what this is supposed to be!